packages/react-components/react-charts-preview/src/components/DonutChart/DonutChartRTL.test.tsx

```diff
@@ -27,15 +28,14 @@ describe('Donut chart interactions', () => {
     const { container } = render(
       <DonutChart data={chartPointsDC} innerRadius={55} calloutProps={{ doNotLayer: true }} />,
     );
-
+    const getByClass = queryAllByAttribute.bind(null, 'class');
     // Act
     const getById = queryAllByAttribute.bind(null, 'id');
     fireEvent.mouseOver(getById(container, /Pie/i)[0]);
-    expect(getById(container, /callout/i)[0]).toBeDefined();
```

Comment:
why remove get by class?

Reply:
After removing the callout id, there's no id present on the DOM itself to locate that particular element, so I resolve this using the getByClass method to locate the element.